x-pack/plugins/security_solution/cypress/tasks/all_cases.ts

```diff
@@ -13,7 +13,7 @@ import {
 import { waitForPageToBeLoaded } from './common';
 
 export const goToCreateNewCase = () => {
-  cy.get(ALL_CASES_CREATE_NEW_CASE_BTN).click({ force: true });
+  cy.get(ALL_CASES_CREATE_NEW_CASE_BTN, { timeout: 60000 }).click({ force: true });
```

Comment:
I was seeing intermittent failures here where it would timeout waiting for the add to new case button. So just bumping the timeout as it eventually shows the button.